### ISSUE TITLE
Attempts to update @polkadot/api to match the version in @plugnet/plug-api-types@1.0.0-rc4 results in BadProof

### ISSUE THREAD

**Issue by hally9k** [plugblockchain/api-template#4]
I tried to update the api-template so that the version of `@plugnet/plug-api-types` matches the version of the `plug-blockchain`, and the version of the `@polkadot/api` matches the one used in `@plugnet/plug-api-types@1.0.0-rc4` - resulting in the following package.json:

```
{
  "name": "plug-api-template",
  "version": "1.0.0",
  "description": "A template project for scripting against the Pl^g blockchain API",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/plugblockchain/api-template.git"
  },
  "author": "Pl^g Developers",
  "license": "GPL-3.0-or-later",
  "bugs": {
    "url": "https://github.com/plugblockchain/api-template/issues"
  },
  "homepage": "https://github.com/plugblockchain/api-template#readme",
  "resolutions": {
    "@polkadot/types": "1.8.1",
    "@polkadot/metadata": "1.8.1",
    "@polkadot/api": "1.8.1",
    "@polkadot/api-derive": "1.8.1",
    "@polkadot/rpc-core": "1.8.1",
    "@polkadot/keyring": "1.8.1"
  },
  "dependencies": {
    "@plugnet/plug-api-types": "git+https://github.com/plugblockchain/plug-api-types.git#1.0.0-rc4",
    "@polkadot/api": "1.8.1"
  }
}
```
I also had to pin the @polkadot/keyring version to get it to run, but I get a BadProof error now:

```
Connecting to ws://localhost:9944
2020-09-23 08:58:19        API/INIT: RPC methods not decorated: state_queryStorageAt}
Unknown signed extensions [OptionSignedExtension] found, treating them as no-efect
Unknown signed extensions [OptionSignedExtension] found, treating them as no-efect
Unknown signed extensions [OptionSignedExtension] found, treating them as no-efect
Unknown signed extensions [OptionSignedExtension] found, treating them as no-efect
You are connected to chain Development using Substrate Node v2.0.0-alpha.5
Alice is transferring 1_500_000_000 to Bob
2020-09-23 08:58:19        RPC-CORE: submitExtrinsic(extrinsic: Extrinsic): Hash:: 1010: Invalid Transaction: BadProof
Error: Error: submitExtrinsic(extrinsic: Extrinsic): Hash:: 1010: Invalid Transaction: BadProof
    at CatchSubscriber.selector (/Users/hally9k/Repos/Trackback/api-template/node_modules/@polkadot/rpc-core/index.js:186:38)
    at CatchSubscriber.error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/operators/catchError.js:48:31)
    at MapSubscriber.Subscriber._error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/Subscriber.js:92:26)
    at MapSubscriber.Subscriber.error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/Subscriber.js:72:18)
    at SwitchMapSubscriber.OuterSubscriber.notifyError (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/OuterSubscriber.js:26:26)
    at InnerSubscriber._error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/InnerSubscriber.js:31:21)
    at InnerSubscriber.Subscriber.error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/Subscriber.js:72:18)
    at CombineLatestSubscriber.OuterSubscriber.notifyError (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/OuterSubscriber.js:26:26)
    at InnerSubscriber._error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/InnerSubscriber.js:31:21)
    at InnerSubscriber.Subscriber.error (/Users/hally9k/Repos/Trackback/api-template/node_modules/rxjs/internal/Subscriber.js:72:18)
```

**Comment by jordy25519** [comment]
Can you re-try the steps by building the plug node using [1.0.0-rc4.2](https://github.com/plugblockchain/plug-blockchain/tree/1.0.0-rc4.2) and pulling master for this project. I've merged #5 with an updated package.json. verified working in my tests

**Comment by hally9k** [comment]
Yep that worked, shot!
